### DISCUSSION TITLE
Issues mapping the autorestic.yml file in a bind mount volume in docker

### DISCUSSION THREAD

**Discussion by miguelarios** [cupcakearmy/autorestic#271] [Q&A] [answered]
I am trying to run this using UnRAID and I am bind mounting 2 folders: 1 data folder to backup and 1 config folder in my appdata folder where all my dockers exist. 

I am unsure where the .autorestic.yml file is suppose to lie. I want the yml file to be in the config folder which is separate from my data folder, but regardless how I name my bind volume mounts i still have issues getting the container to run. It auto shutsdowns.

**Comment by miguelarios** [comment, marked as answer]
Ok so this is no longer an issue as I went ahead and stopped using the docker image and chose to directly install autorestic. It seems that the docker version of this is a container meant to be used with the docker run command as a one time use to backup. I preferred a persistent install so not using the docker method anymore. I think this is why the container wasn't continuously running as the entrypoint and CMD in the dockerfile doesn't run a continuous process.

**Comment by Absentity** [comment]
I store my config in `/mnt/user/appdata/autorestic/`.

I have the following plugins installed as well:
- Unassigned Devices
- User Scripts

In User Scripts, you can create a script and have it run at custom intervals. This is my script for backups:
```sh
docker run --rm \
  -v /mnt/user/appdata/autorestic:/data \
  -v /mnt/user:/mnt/user \
  -v /mnt/disks/Backup:/mnt/disks/Backup \
  cupcakearmy/autorestic \
  autorestic backup -va -c /data/.autorestic.yaml
```
Note: `/mnt/disks/Backup` is an unassigned device, and I *think* this directory may be changing to `/mnt/remotes/` in case this doesn't work for you.

This does create an orphaned image entry in your Docker tab in UnRAID, and you do need to download the image originally and keep it up to date.
